src/pages/search.vue

```diff
@@ -1,6 +1,7 @@
 <template>
   <VSkipToContentContainer
-    class="browse-page flex flex-col w-full px-4 md:px-10"
+    class="browse-page flex flex-col px-4 md:px-10"
+    :class="[isFilterSidebarVisible ? 'w-full' : 'w-screen']"
```

Comment:
At least on Firefox for Linux, the `w-screen` seems like it adds horizontal scroll on the search results when the filter sidebar isn't open :thinking:

Reply:
I can confirm this. Also, when the sidebar is closed, it still has a width of 1px because it still has a `border-start`.